### PR TITLE
SUBMARINE-504. Enable angular style check in github actions

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
- Enable angular style check in github actions
- Fix code style in workbench-ng

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-504

### How should this be tested?
https://github.com/pingsutw/hadoop-submarine/actions/runs/107861642
https://travis-ci.org/github/pingsutw/hadoop-submarine/builds/688248808

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-workbench/workbench-web-ng/src/app/pages/user/login/login.component.ts [hunk 1]
```diff
@@ -74,11 +74,12 @@ export class LoginComponent implements OnInit {
   }
 
   requestFailed(error: Error) {
-    this.nzNotificationService.error('Login Failed',
-      'Username and password are incorrect, ' +
-          'please try again or create an account',
+    this.nzNotificationService.error(
+      'Login Failed',
+      'Username and password are incorrect, ' + 'please try again or create an account',
```
Comment (liuxunorg) [inline submarine-workbench/workbench-web-ng/src/app/pages/user/login/login.component.ts:79]: The plus sign here can be deleted

**Comment by liuxunorg** [comment]
I have a suggestion: add the `style_check` directory under the `/submarine/dev-support` directory, and then put `lint-java.sh`, `lint-scala.sh` and `lint-angular.sh` into it.

File: submarine-workbench/workbench-web-ng/src/app/pages/user/login/login.component.ts (hunk 1 above)
Comment (pingsutw) [reply submarine-workbench/workbench-web-ng/src/app/pages/user/login/login.component.ts]: Good catch. done

**Comment by pingsutw** [comment]
@liuxunorg Thanks for the review.  I've updated PR

**Pull request closed without merging**